### DISCUSSION TITLE
How to save / serialize the data stored in MemoryStore when server shutting down and load / deserialize it back in when server start ?

### DISCUSSION THREAD

**Discussion by lmezg** [salvo-rs/salvo#421] [Q&A]
I didn't find any function currently implemented in salvo, I think this would be useful to preserve the session data when restarting server. Would you please show me how this should be done ?

**Comment by chrislearn** [comment]
This is a uncommon function requirement.

**Comment by lmezg** [comment]
yeah, it's possible this is not common since i'm not familiar with the web server. I wonder how others save this session related data (using MemoryStore) when they restart the server if they're not using redis ?
